### PR TITLE
add logging to session enumeration, port connection_info to pvHelpers

### PR DESCRIPTION
**Pull request by samantehrani**

### CODE REVIEW

File: pvHelpers/__init__.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from .hook_decorators import *
 from .application_config import ApplicationConfig
 from .proxy_config import *
+from .connection_info import *
```
Comment (hu13) [inline pvHelpers/__init__.py]: this is not being used right given https://github.com/PreVeil/core/pull/1649/files#diff-c939b7c062708b34ff1a3098758416cdR1379 is commented out?

File: pvHelpers/win_helpers.py [hunk 2]
```diff
@@ -596,76 +596,71 @@ def __del__(self):
         super(WinPopen, self).__del__()
 
 
+def list_active_sessions():
+    return filter(
+        lambda s: s["State"] == win32ts.WTSActive and s["SessionId"] != 0,
```
Comment (hu13) [inline pvHelpers/win_helpers.py]: is there a Microsoft doc for this? `s["SessionId"] != 0` means?

File: pvHelpers/__init__.py (hunk 1 above)
Comment (samantehrani) [reply pvHelpers/__init__.py]: coorect

File: pvHelpers/win_helpers.py (hunk 2 above)
Comment (samantehrani) [reply pvHelpers/win_helpers.py]: sorry, will leave comment

**Pull request merged**